src/protocols/vnc/settings.c

```diff
@@ -66,6 +66,7 @@ const char* GUAC_VNC_CLIENT_ARGS[] = {
     "sftp-private-key",
     "sftp-passphrase",
     "sftp-directory",
+    "sftp-keepalive",
```

Comment:
I suggest finding consistent wording between the `sftp-` parameters and those used by the SSH client. Here, things are referred to as `keepalive`, while the SSH client uses `server-alive-interval`. One or the other should be altered such that things are predictable.